### PR TITLE
Fix warnings from CI Build

### PR DESCRIPTION
**Pull request by frank-dong-ms**
Fix/suppress all warnings from CI build:
1. Skip some dll copy at build stage on CI build, this copy is only necessary for official build
2. Suppress NU5118 warning for all projects that generating nuget packages. mlnet has its own logo file and the warning is complaining about mlnet logo file and the default logo file shares same PackagePath, we can just ignore this warning.
3. Suppress NU5127 and NU5128 for Microsoft.ML project. These warnings are complaining about missing some default folder structure or file from Microsoft.ML nuget package. Microsoft.ML package is built from several projects and these missing default folder and file are actually present but nuget seems not able to correctly see this condition so ignore these warnings.
4. Remove unnecessary stage from CI pipeline.

### CODE REVIEW

File: src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.Extensions.ML</IncludeInPackage>
+    <NoWarn>$(NoWarn);NU5118</NoWarn>
```
Comment (harishsk) [inline src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj:7]: Can this property be included in some top level Directory.Build.props?

Comment (frank-dong-ms) [reply src/Microsoft.Extensions.ML/Microsoft.Extensions.ML.csproj:7]: Yes, that is great idea, I think we can.

---
In reply to: [538041772](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538041772) [](ancestors = 538041772)

File: build/ci/job-template.yml [hunk 2]
```diff
@@ -148,16 +148,6 @@ jobs:
     - script: $(dotnetPath) msbuild -restore build/Codecoverage.proj
       displayName: Upload coverage to codecov.io
       condition: and(succeeded(), eq(${{ parameters.codeCoverage }}, True))
-    - task: PublishTestResults@2
```
Comment (eerhardt) [inline build/ci/job-template.yml:151]: Don't we want this? I don't see the `Test` tab on the CI build in this PR:

image

Publishing the test results allows Azure DevOps to easily show test failures:

image

Comment (frank-dong-ms) [reply build/ci/job-template.yml:151]: after migrate to arcade we are not generating any trx test results so this step is not necessary any more. we have a stage (the next one) that upload test results (html format) if test fails.

---
In reply to: [538723243](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538723243) [](ancestors = 538723243)

Comment (harishsk) [reply build/ci/job-template.yml:151]: I think Eric's point maybe that Azure has the capability to show test failures right away without having to scan through the build logs. We aren't generating trx test results, but maybe Arcade has a different mechanism to enable this. Can you please look into it?

---
In reply to: [538732212](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538732212) [](ancestors = 538732212,538723243)

Comment (frank-dong-ms) [reply build/ci/job-template.yml:151]: Sure, I will take a look, that can be done in separate PR.

---
In reply to: [538734618](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538734618) [](ancestors = 538734618,538732212,538723243)

**Review by harishsk — APPROVED** [review]
:shipit:

File: build/ci/job-template.yml (hunk 2 above)
Comment (harishsk) [reply build/ci/job-template.yml:151]: Sounds good.

---
In reply to: [538752000](https://github.com/dotnet/machinelearning/pull/5541#discussion_r538752000) [](ancestors = 538752000,538734618,538732212,538723243)

**Pull request merged**